plugins/audio-buffer.cc

```diff
@@ -8,6 +8,7 @@ namespace clap {
    void AudioBuffer<T>::fromClap(const clap_audio_buffer *buffer,
                                  uint32_t frameOffset,
                                  uint32_t frameCount) noexcept {
+      assert(frameOffset + frameCount <= _frameCount);
```

Comment:
This assertion is wrong.
frameOffset + frameCount is only relevant for the clap buffer

Reply:
_frameCount is the nuber of samples in this buffer. 
frameOffset+frameCount is the highest index accessed +1.

Reply:
frameOffset is for the clap buffer, not AudioBuffer